### PR TITLE
Admin trips index

### PR DESCRIPTION
**Pull request by anon0mys**
###### Changes Made:
* Admin can edit a trip
* Admin can delete a trip
* Added buttons to trips show for edit and delete
* Namespacing for admin trip routes
* Added admin trips controller

### CODE REVIEW

File: app/controllers/admin/trips_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Admin::TripsController < Admin::BaseController
+  def edit
+    @trip = Trip.find(params[:id])
+  end
+
+  def update
+    trip = Trip.find(params[:id])
+    if trip.update(trip_params)
+      flash[:success] = "Trip #{trip.id} was updated"
```
Comment (anon0mys) [inline app/controllers/admin/trips_controller.rb]: I would like to put together a list of css for flash messages. Any ideas?

File: app/views/admin/trips/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<%= form_for [:admin, @trip] do |f| %>
+  <div class=''>
```
Comment (anon0mys) [inline app/views/admin/trips/_form.html.erb]: I built this form as a series of divs. Is everyone ok with using bootstrap form-controls for styling?

File: app/controllers/admin/trips_controller.rb (hunk 1 above)
Comment (hmesander) [reply app/controllers/admin/trips_controller.rb]: That can be easily added as the css classes are already built-in (.notice, .success, etc).  I will open a chore in pivotal for that.

File: app/views/admin/trips/_form.html.erb (hunk 2 above)
Comment (jrambold) [reply app/views/admin/trips/_form.html.erb]: We'll probably want to go trough and look at styling as I don't think we have this throughout.
We can add a story for standardizing styling

Comment (jamesrnelson) [reply app/views/admin/trips/_form.html.erb]: I've got a table for the forms right now, but we can merge this now and decide later.

Comment (hmesander) [reply app/views/admin/trips/_form.html.erb]: I'm cool with bootstrap form-controls for styling these forms.

**Pull request merged**